### ISSUE TITLE
Publish Maven artifact to Central

### ISSUE THREAD

**Issue by fpaaske** [robotmedia/AndroidBillingLibrary#98]
It would be very useful if you could make an official maven artifact available through Central.
Currently, using ABL in a Maven project breaks the flow (requires code embedding or "system scope" dependency).

Thanks!

**Comment by hpique** [comment]
Happy to do that.

I'm totally new to Maven. Can you give me some pointers/links?

**Comment by fpaaske** [comment]
Excellent! I'm mostly familiar with Maven through the usage of existing libraries, so I cannot help much but to point you to their [web page](http://maven.apache.org/users/index.html). Perhaps particularly to [this section](http://maven.apache.org/repository/index.html).

Edit: Sorry, accidentally clicked close instead of just comment :-)

**Comment by tjerkw** [comment]
So is this already in a maven repository?

**Comment by hpique** [comment]
@tjerkw Nope. Couldn't find time to dive in to Maven yet. If someone wants to help I'm totally open to it.

**Comment by tjerkw** [comment]
I found the for RoboBillingLibrary which also has support for Amazon billing. It is also mavenized. Not in a central repo, but easy to install in your own repo:

https://github.com/christopherperry/RoboBillingLibrary

Maybe the pom.xml files from that library could help.

Cool project btw!
